### ISSUE TITLE
Verify contacts by QR code / key fingerprint

### ISSUE THREAD

**Issue by pingberlin** [blizzard4591/openMittsu#26]
Is there any possibility to verify contacts by QR code or by manually verifying their "key fingerprint" in the contact details?

Currently under "Edit Contact" in openMittsu there's only a "Public Key:" entry. But in Threema there's only a "key fingerprint" entry. So there seems to be no way to verify.

**Comment by blizzard4591** [comment]
No, this is not yet supported in the currently released version (mainly because the contact storage does not contain data on "trust").

This can be changed soon when I finally release the database-backed next version of the client :)
